src/framework/extensions/internal/extensionsuiactions.cpp

```diff
@@ -38,11 +34,11 @@ static UiAction MANAGE_ACTION = UiAction(
     "manage-plugins",
     mu::context::UiCtxAny,
     mu::context::CTX_ANY,
-    TranslatableString("action", "&Manage plugins…"),
-    TranslatableString("action", "Manage plugins…")
+    mu::TranslatableString("action", "&Manage plugins…"),
+    mu::TranslatableString("action", "Manage plugins…")
```

Comment:
should we rename it to Manage extensions?

Reply:
I asked Bradley..